src/services/CertificateGenerationService.ts

```diff
@@ -95,6 +95,7 @@ class CertificateGenerationService {
       trl_fail: config.documentNames.vtg30,
       trl_fail_bilingual: config.documentNames.vtg30_bilingual,
       trl_prs: config.documentNames.trl_prs,
+        trl_prs_bilingual: config.documentNames.trl_prs_bilingual,
```

Comment:
For some reason it's moved this over 2 spaces, can you fix the formatting please (it did the same for me)